shootout/shootout.py

```diff
@@ -0,0 +1,385 @@
+
+    async def get_random_wait_time(self, ctx) -> int:
+        delay = await self.config.guild(ctx.guild).Times.delay()
+        fuzzy = await self.config.guild(ctx.guild).Times.fuzzy()
+        return random.randint(delay - fuzzy, delay + fuzzy)
+
+    async def get_session_from_context(self, ctx):
+        channel_id = ctx.channel.id
+        sessions = await self.config.guild(ctx.guild).Sessions()
+        # Check through all of our currently loaded sessions
+        for session in sessions:
+            if session["Channel"] == channel_id:
+                return session
+        # Session for this channel does not exist
+        new_session = self.session_config.copy()
+        new_session["Channel"] = channel_id
+        sessions.append(new_session)
+        await self.config.guild(ctx.guild).Sessions.set(sessions)
+        return new_session
+
+    async def save_session_to_config(self, ctx, the_session) -> None:
+        sessions = await self.config.guild(ctx.guild).Sessions()
+        for index, session in enumerate(sessions):
+            if session["Channel"] == the_session["Channel"]:
+                sessions[index] = the_session
+        await self.config.guild(ctx.guild).Sessions.set(sessions)
+
+    async def clear_session(self, ctx) -> None:
+        sessions = await self.config.guild(ctx.guild).Sessions()
+        for index, session in enumerate(sessions):
+            if session["Channel"] == ctx.channel.id:
+                sessions.remove(session)
+        await self.config.guild(ctx.guild).Sessions.set(sessions)
+
+    @commands.command()
+    @commands.guild_only()
+    async def shootout(self, ctx):
+        """Start or join a shootout!
+
+        A shootout is a game where players can join by putting up money into the pot (if configured).
+
+        The bot will send a message to chat saying to begin and what message to type to win.
+
+        First person who types the message wins the entire pot! (Even if that pot is 0).
+
+        The game will not start if no other players have joined, and any cost will be refunded.
+
+        There is no maximum number of players.
+        """
+        settings = await self.config.guild(ctx.guild).all()
+        if await self.game_checks(ctx, settings):
+            await self.add_player(ctx, settings["cost"])
+
+    @commands.group()
+    @commands.guild_only()
+    @checks.mod_or_permissions(manage_guild=True)
+    async def soset(self, ctx):
+        """Change settings to Shootout"""
+        pass
+
+    @soset.command(name="bet")
+    async def soset_bet(self, ctx, amount: int):
+        """Sets the bet amount for a shootout"""
+        if amount < 0:
+            return await ctx.send("Get your negativity out of here!")
+        await self.config.guild(ctx.guild).cost.set(amount)
+        await ctx.tick()
+        return
+
+    @soset.command(name="draw")
+    async def soset_draw(self, ctx, *, draws: str = None):
```

Comment:
Since victories has add/remove/list commands, you should do the same thing for draws.